fluent-plugin-mysql.gemspec

```diff
@@ -14,9 +14,10 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ["lib"]
 
-  gem.add_runtime_dependency "fluentd", '~> 0.12.0'
+  gem.add_runtime_dependency "fluentd", ['>= 0.14.8', '< 2']
```

Comment:
Yes.
Fluentd 0.14.4, 0.14.5 has in_tail crash bug. 0.14.7 has `chunk#msgpack_each` bug in compatibility layer.